content/yaml-quick-start/building-a-unity-app.md

```diff
@@ -0,0 +1,492 @@
+---
+title: Building Unity mobile apps
+description: How to build Unity mobile apps with codemagic.yaml
+weight: 12
+
+---
+
+Unity is a cross-platform game engine developed by Unity Technologies. It can be used to create mobile applications that run on iOS and Android.
+
+Codemagic's Unity premium macOS base image has the Unity SDK, iOS and Android modules pre-installed. 
+
+This guide will show you how to configure a workflow that builds and publishes your app to TestFlight and Google Play using a **codemagic.yaml** configuration file. 
+
+{{<notebox>}}
+**Important!** Building Unity apps on Codemagic is currently in closed beta release. If you are interested in taking part in the beta, please contact us [here](https://codemagic.io/contact/)
+{{</notebox>}}
+
+## Adding your Unity project to Codemagic
+
+The apps you have available on Codemagic are listed on the Applications page. Click **Add application** to add a new app.
```

Comment:
Perhaps link to the relevant docs about adding new apps so as to skip this part here